spec/classes/ccs/graphical_spec.rb

```diff
@@ -22,30 +22,30 @@
 
       it { is_expected.to compile.with_all_deps }
 
-      include_examples 'x2go packages', os_facts: os_facts
+      include_examples('x2go packages', os_facts:)
```

Comment:
oh is that valid syntax, `os_facts:`?

Reply:
That was a rubocop refactor when I raised the target ruby version.